src/v2/view-builder/views/PollView.js

```diff
@@ -43,17 +47,34 @@ const Body = BaseForm.extend(Object.assign(
 
     remove() {
       BaseForm.prototype.remove.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
     },
 
     triggerAfterError() {
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
-      this.stopPolling();
+      clearTimeout(this.refreshTimeout);
+      this.stopCountDown();
       this.$el.find('.o-form-fieldset-container').empty();
     },
-  },
 
-  polling,
+    startCountDown(selector , interval) {
+      if (this.countDown) {
+        clearInterval(this.countDown);
+      }
+      this.counterEl = this.$el.find(selector);
+      this.countDown = setInterval(() => {
+        if(this.counterEl.text() !== '0') {
```

Comment:
It was intentionally checking for `this.counterEl.text() !== '1'` .. i think it adds an extra second or shows `0` seconds remaining otherwise ??

Reply:
yeah this is intended and per v1. In case the actual request is taking longer the user should see 0 seconds and spinner and not stuck in 1 secopnd.